packages/server/src/blitz-version.ts

```diff
@@ -0,0 +1,32 @@
+import {pathExists, readFile, writeFile} from "fs-extra"
+import {resolve} from "path"
+
+export function getBlitzVersion(): string | null {
+  try {
+    const path = require.resolve("blitz/package.json")
+    const pkgJson = require(path)
+    return pkgJson.version
+  } catch {
+    return null
+  }
+}
+
+export async function isVersionMatched(buildFolder: string = ".blitz/caches") {
+  const versionStore = resolve(buildFolder, "blitz-version")
```

Comment:
1. I think maybe `blitz-version.txt` would be better?
2. There's a couple places in this file where you use the file name. Can you extract that out into a constant?

Reply:
1. I agree with that
2. do you need me to define it on the same file or different file?

Reply:
same file is fine